src/main/kotlin/org/kamiblue/client/module/modules/combat/AutoMend.kt

```diff
@@ -113,6 +142,14 @@ internal object AutoMend : Module(
         }
     }
 
+    private fun SafeClientEvent.findEmptySlot(min: Int): Int {
+        for(i in min..36) {
+            if(player.inventory.getStackInSlot(i).isEmpty)
+                return i
+        }
+        return -1
+    }
+
```

Comment:
`player.storageSlot.firstEmpty`

Reply:
I know but I would rather use this method because 1 tick is needed for the inventory to be updated

Reply:
It is just the same thing but more clean

Reply:
if multiple armor pieces have 100% durability, player.storageSlot.firstEmpty won't work because AutoMend tries to move multiple armor pieces to one slot

Reply:
(in 1 tick)